modules/distillery.cpp

```diff
@@ -529,7 +529,7 @@ void LapH::distillery::add_to_perambulator(const size_t dil_t, const size_t dil_
     exit(0);
   }
 
-  Eigen::setNbThreads(1);
+  //Eigen::setNbThreads(1);
```

Comment:
this is not needed anymore, because set in `main`?

Reply:
Yeah, we can keep it here though to make double-sure.